Contributor updates on repo-specific documentation (#83)
NCAR/geocat-comp: README.md

The term "GeoCAT-comp" stands for both the whole computational component of the [GeoCAT](https://ncar.github.io/GeoCAT) 
project and a single Github repository as described here. As the computational component of 

> [GeoCAT](https://ncar.github.io/GeoCAT), GeoCAT-comp wraps NCL's non-WRF Fortran routines into Python.

Reviewer: Perhaps we should generalize this a bit so it is less NCL-centric, and say something like "GeoCAT-comp provides implementations of computational functions for operating on geosciences data. Many of these functions originated in NCL...

Author: Agreed and addressed now.

> The term "GeoCAT-comp" stands for both the whole computational component of the [GeoCAT](https://ncar.github.io/GeoCAT) 

Reviewer 2: > The term "GeoCAT-comp" stands for both...

The wording here feels a little funny. Maybe something like 
 > GeoCAT-comp is both the whole ...

would sound smoother?

Author: Thanks for this suggestion! Addressed it.

project and a single Github repository as described here. As the computational component of 
[GeoCAT](https://ncar.github.io/GeoCAT), GeoCAT-comp provides implementations of computational functions for operating 
on Geosciences data. Many of these functions originated in NCL are pivoted into Python with the help of GeoCAT-comp; 

> however, developers are welcome to come up with novel computational functions for Geoscience data.

Reviewer 2: I think geoscience might need to be uncapitalized in this line.

Author: Thanks for this! Addressed.

![actions](https://github.com/NCAR/geocat-comp/workflows/actions/badge.svg)
[![Documentation Status](https://readthedocs.org/projects/geocat-comp/badge/?version=latest)](https://geocat-comp.readthedocs.io/en/latest/?badge=latest)

GeoCAT-comp is both the whole computational component of the [GeoCAT](https://ncar.github.io/GeoCAT) 
project and a single Github repository as described here. As the computational component of 
[GeoCAT](https://ncar.github.io/GeoCAT), GeoCAT-comp provides implementations of computational functions for operating 

> on geosciences data. Many of these functions originated in NCL are pivoted into Python with the help of GeoCAT-comp; 

Reviewer: ```suggestion
on geosciences data. Many of these functions originated in NCL and were translated into Python with the help of GeoCAT-comp; 
```
